<!-- -->
<launch>
  <arg name="role_name" default="ego_vehicle" />

  <node pkg="psaf_steering" type="carla_twist_pid_control_node.py" name="psaf_twist_p_control_$(arg role_name)" output="screen">
    <param name="role_name" value="$(arg role_name)" />
  </node>
</launch>
